src/components/pushClient/index.js

```diff
@@ -21,8 +21,8 @@ export const isIOS = () =>
 export const DESKTOP_BANNER = 'desktop_banner'
 export const NOVIEWER_DESKTOP_CTA = 'noviewer_desktop_cta'
 
-export const isClientAlreadyInstalled = async () => {
-  const resp = await cozy.client.fetchJSON('GET', '/settings/clients')
+export const isClientAlreadyInstalled = async client => {
+  const resp = await client.query(client.get('io.cozy.settings', 'clients'))
```

Comment:
No tests seem to have been changed by this change, I think it would be a good time to add one, what do you think ?

Reply:
Done in #2043